packages/translation/src/lang/cn.json

```diff
@@ -3549,7 +3549,8 @@
           "restarting": "正在重启",
           "exited": "已退出",
           "removing": "删除中",
-          "dead": "废弃"
+          "dead": "废弃",
+          "orphaned": "孤立"
```

Comment:
Please revert all translation changes except English, they are provided by our community and don't need to be added via PR